frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -540,7 +540,7 @@ describe("Where's My Gene", () => {
         validationMessage.getByText(FAILED_EMAIL_VALIDATION_STRING)
       ).toBeTruthy();
 
-      emailInput.fill("");
+      emailInput.fill(" ");
```

Comment:
What's the reason for an empty space vs. empty string?

Reply:
@kaloster when you get a chance PTAL thank you!

Reply:
Yeah that was a temp fix that I did for a test that was failing after the refactor but I have since found the culprit and this is no longer needed. Here's a PR that reverts it back to an empty string: https://github.com/chanzuckerberg/single-cell-data-portal/pull/6011